src/excmds.ts

```diff
@@ -3084,6 +3097,15 @@ export async function bmark(url?: string, ...titlearr: string[]) {
         }
         const validpaths = flatten(treeClimber(tree, "")).filter(x => "path" in x)
         title = title.substring(title.lastIndexOf("/") + 1)
+        //technically an initial title string like `Firefox/` can give us a blank title
+        //once we remove the path, so let's fix that
+        if (title == "") {
+            //retrieve title from current tab
+            var currTitle
+            let currTitle = (await activeTab()).title
```

Comment:
These three lines can just be replaced with `title = (await...`. currTitle doesn't do anything.